### PR TITLE
LPD-23437 - poshi -Rename "Fields" to "Visualization Modes"

### PR DESCRIPTION
**Pull request by kresimir-coko**

### CODE REVIEW

**Comment by kresimir-coko** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/frontendinfrastructure/DataSetAdmin.macro [hunk 1]
```diff
@@ -324,7 +324,7 @@ definition {
 			key_viewName = ${dataSetViewName});
 
 		if (isSet(key_fieldList)) {
-			DataSetAdmin.goToTab(tabName = "Fields");
+			DataSetAdmin.goToTab(tabName = "Visualization Modes");
```
Comment (markocikos) [inline portal-web/test/functional/com/liferay/portalweb/macros/frontendinfrastructure/DataSetAdmin.macro:327]: I don't think we have any list and cards poshi tests, so this should be enough, `Table` is the default inner tab.

**Review by markocikos — APPROVED** [review]
@kresimir-coko LGTM! :sparkles:

**Comment by kresimir-coko** [comment]
Hey @markocikos I added a couple more commits
- https://github.com/liferay-frontend/liferay-portal/pull/4160/commits/dd5a39872fb65023e0bdd27b040cdd5fa455b84f - Removing Poshi tests that I deemed were duplicates of Playwright ones (this commit might be better done in a separate PR, for simplicity's sake)
- https://github.com/liferay-frontend/liferay-portal/pull/4160/commits/5b8f6f1204638597eed1e1ac8c256432896898ad - Rename `Fields` to `Visualization Modes` in `*.macro` files

**Comment by kresimir-coko** [comment]
ci:test:sf

**Comment by kresimir-coko** [comment]
ci:test:relevant

**Comment by kresimir-coko** [comment]
Reopened in https://github.com/liferay-frontend/liferay-portal/pull/4167 due to CI being stuck

**Comment by markocikos** [comment]
@kresimir-coko you don't need to open a new PR to restart CI, just typing `ci:*` here is enough.

**Pull request closed without merging**